rpmbuild/SPECS/hilbert-cli.spec

```diff
@@ -18,14 +18,17 @@ Name:           hilbert-cli
 Version:        0.9.1
 
 License:        Apache License, Version 2.0
-Release:        7%{?dist}
+Release:        8%{?dist}
 
 URL:            https://github.com/hilbert/%{origname}
-Source0:         hilbert-cli.tar.gz
+Source0:        https://cloud.imaginary.org/index.php/s/WSGU4yEaR4RaH3T/download?path=%2F&files=hilbert-cli.tar.gz
```

Comment:
Why are you linking to the cloud? What's in the tar.gz? Why not use https://github.com/hilbert/hilbert-cli/archive/devel.tar.gz (similar for any other branch or tag)?

Reply:
Currently it is generated via `git archive -9 --format tar.gz -o hilbert-cli.tar.gz HEAD`. 
Cloud is something i can easily control. no time to experiment with rpm building :-(

Reply:
But since both archives should share the same content, I suppose you could just replace the url to point to github.

Reply:
hm, we can change it officially.... but for generating ALL preliminary RPMs i will always have to change the spec file locally in order to use the current (preliminary) sources :-(

Reply:
should i just remove remote URLs and use local tarballs instead as it was before?